x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/index_view_logic.test.ts

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { LogicMounter, mockFlashMessageHelpers } from '../../../__mocks__/kea_logic';
+
+import { apiIndex, connectorIndex } from '../../__mocks__/view_index.mock';
+
+import { nextTick } from '@kbn/test-jest-helpers';
+
+import { HttpError } from '../../../../../common/types/api';
+
+import { StartSyncApiLogic } from '../../api/connector_package/start_sync_api_logic';
+import { FetchIndexApiLogic } from '../../api/index/fetch_index_api_logic';
+
+import { IngestionMethod, IngestionStatus } from '../../types';
+
+import { indexToViewIndex } from '../../utils/indices';
+
+import { IndexViewLogic } from './index_view_logic';
+
+const DEFAULT_VALUES = {
+  data: undefined,
```

Comment:
I'm surprised these can be undefined I thought kea prevented setting values to undefined (aka why we have "| null" everywhere)

Reply:
They can't, this object is just untyped /shrug